de/manual/ch_Finance-Quote.xml

```diff
@@ -174,11 +174,11 @@ SYNOPSIS
       </step>
 
       <step>
-        <simpara>Führen Sie den Befehl <userinput>gnc-fq-check</userinput> aus, um zu überprüfen, ob das Programm
+        <simpara>Führen Sie den Befehl <userinput>gnucash-cli --quotes info</userinput> aus, um zu überprüfen, ob das Programm
```

Comment:
Perhaps we should keep the old commands here and later in a footnote?
Before version … one had to use …

Reply:
In my opinion, the manual describes the current version.
I, as a user, do not expect the description of old functions. If I still work with version 4, then I have to look into the corresponding manual, which is still available on the homepage.

Reply:
I agree with @CWehli on this one.

Reply:
But as a long time user I'm used to enter gnc-fq-… and now I get command-not-found. Then I expect an answer from the recent manual when I search it for `gnc-fq`.  A footnote should tell me `In version x.y gnc-fq-… was removed. Use gnucash-cli … instead. `